app/src/main/java/com/quran/labs/androidquran/ui/fragment/TabletFragment.java

```diff
@@ -319,16 +320,18 @@ protected void onPostExecute(RectF[] rect) {
           if (mRightImageView != null && mLeftImageView != null) {
             mRightImageView.setPageBounds(rect[0]);
             mLeftImageView.setPageBounds(rect[1]);
-            Context context = getContext();
-            if (mOverlayText && context != null) {
```

Comment:
I see. 

I think, for that scenario, checking if fragment is being removed ([`isRemoving()`](https://developer.android.com/reference/android/app/Fragment.html#isRemoving())) or if it is visible ([`isVisible()`](https://developer.android.com/reference/android/app/Fragment.html#isVisible())) would fit better.